blocks/library/latest-posts/block.js

```diff
@@ -49,15 +49,16 @@ class LatestPostsBlock extends Component {
 
 	render() {
 		const latestPosts = this.props.latestPosts.data;
-		const { attributes, isSelected, setAttributes } = this.props;
+		const { attributes, categoriesList, isSelected, setAttributes } = this.props;
 		const { displayPostDate, align, layout, columns, order, orderBy, categories, postsToShow } = attributes;
 
 		const inspectorControls = isSelected && (
 			<InspectorControls key="inspector">
 				<h3>{ __( 'Latest Posts Settings' ) }</h3>
-				<QueryPanel
+				<QueryControls
 					{ ...{ order, orderBy } }
 					numberOfItems={ postsToShow }
+					categories={ get( categoriesList, 'data', {} ) }
 					category={ categories }
```

Comment:
Shouldn't `category` be renamed to `selectedCategoryId` in here to reflect its further usage? Should we also rename `categories` to something that is closer to the aforementioned prop name?

Reply:
The props were renamed :)